### PR TITLE
docs(examples): Add routes-gen example

### PR DESCRIPTION
**Pull request by sandulat**

### CODE REVIEW

File: examples/routes-gen/app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        {process.env.NODE_ENV === "development" && <LiveReload />}
```
Comment (MichaelDeBoey) [inline examples/routes-gen/app/root.tsx:23]:
```suggestion
        <LiveReload />
```
See #1352

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by kentcdodds — CHANGES_REQUESTED** [review]
This looks stellar! Just the one thing as noted by @MichaelDeBoey and we'll be good to go 👍

**Comment by sandulat** [comment]
Thank you @kentcdodds @MichaelDeBoey!

**Review by kentcdodds — APPROVED** [review]
Sweet!

**Pull request merged**